### PR TITLE
Typo "Teams Room Pro"→"Teams Rooms Pro"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/devices/smartvision-admin 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/public/Teams/devices/smartvision-admin.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/devices/smartvision-admin.md [hunk 1]
```diff
@@ -41,19 +41,19 @@ The getting started kit for IntelliFrame provides Microsoft Teams certified came
 
 ### Prerequisites
 
-- Microsoft Teams Room Windows device or console.
-- Microsoft Teams Room Pro license:
-  - Microsoft Teams Room with Pro or Premium license is required to enable IntelliFrame, and people recognition features on Microsoft Teams Room.
-  - Basic license won't enable IntelliFrame or people recognition. If you have Teams Room Basic license, the camera shows only active speaker and panoramic views.
+- Microsoft Teams Rooms Windows device or console.
+- Microsoft Teams Rooms Pro license:
+  - Microsoft Teams Rooms with Pro or Premium license is required to enable IntelliFrame, and people recognition features on Microsoft Teams Room.
```
Comment (JohanFreelancer9) [inline Teams/devices/smartvision-admin.md:46]:
```suggestion
  - Microsoft Teams Rooms with Pro or Premium license is required to enable IntelliFrame, and people recognition features on Microsoft Teams Rooms.
```

File: Teams/devices/smartvision-admin.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Connect the USB-C cable to USB-A cable with the USB-C port inserted into the Sma
   > [!Note]
   > Ensure the cable is not pulled tightly or pinched for ideal data transmission and connectivity.
 
-Log in to the Microsoft Teams Room as an admin user and change the settings as shown in the image.
+Log in to the Microsoft Teams Rooms as an admin user and change the settings as shown in the image.
```
Comment (JohanFreelancer9) [inline Teams/devices/smartvision-admin.md:77]:
```suggestion
Log in to Microsoft Teams Rooms as an admin and change the settings as shown in the image.
```

File: Teams/devices/smartvision-admin.md [hunk 3]
```diff
@@ -84,22 +84,22 @@ Log in to the Microsoft Teams Room as an admin user and change the settings as s
 
 To enable device log collection for feedback log filing:
 
-1. Launch an admin command prompt in Microsoft Teams Room to which the camera is connected.
+1. Launch an admin command prompt in Microsoft Teams Rooms to which the camera is connected.
 2. Run [plazacfg.bat]{.underline} released with this document to config the log collection and windows driver update endpoint settings properly.
-3. Reboot the Microsoft Teams Room for changes to take effect.
+3. Reboot the Microsoft Teams Rooms for changes to take effect.
```
Comment (JohanFreelancer9) [inline Teams/devices/smartvision-admin.md:89]:
```suggestion
3. Restart Microsoft Teams Rooms for changes to take effect.
```

File: Teams/devices/smartvision-admin.md [hunk 4]
```diff
@@ -84,22 +84,22 @@ Log in to the Microsoft Teams Room as an admin user and change the settings as s
...
-To ensure you have the latest firmware, sign in to Microsoft Teams Room from the Admin mode, and do a **Windows update** to receive the latest firmware.
+To ensure you have the latest firmware, sign in to Microsoft Teams Rooms from the Admin mode, and do a **Windows update** to receive the latest firmware.
 
 ## Calibrate to Ignore room monitors
 
 You must calibrate the device to avoid camera detecting faces on monitors and rendering them on IntelliFrame.
 
 1. To complete the calibration process (Automatic and Manual), [download the application](https://agent.rooms.microsoft.com/files/agent/plaza/builds/prod/IntelligentIgnore.zip).
-2. Sign in to Microsoft Teams Room with admin privilege, right select **install.bat** and run as administrator. The app starts the installation.
+2. Sign in to Microsoft Teams Rooms with admin privilege, right select **install.bat** and run as administrator. The app starts the installation.
```
Comment (JohanFreelancer9) [inline Teams/devices/smartvision-admin.md:102]:
```suggestion
2. Sign in to Microsoft Teams Rooms with admin privileges, right-select **install.bat**, and run as an administrator. The app starts the installation.
```

File: Teams/devices/smartvision-admin.md [hunk 5]
```diff
@@ -120,7 +120,7 @@ You must calibrate the device to avoid camera detecting faces on monitors and re
 
 6. Once you set the ignore region, the SmartVision 60 will reboot and the changes will be applied.
```
Comment (JohanFreelancer9) [inline Teams/devices/smartvision-admin.md:121]:
```suggestion
6. Once you set the ignore region, the SmartVision 60 will restart and the changes will be applied.
```

File: Teams/devices/smartvision-admin.md [hunk 6]
```diff
@@ -169,11 +169,11 @@ In some states, people recognition can't be used.
...
 
  ```roomPeopleNameUserOverride``` = {On | Off}  
-**On** - Policy value allow **People recognition** option on Microsoft Teams Room under call control bar.  
+**On** - Policy value allow **People recognition** option on Microsoft Teams Rooms under call control bar.  
 **Off** – No People Recognition option on Microsoft Teams Room. This is default.
 
 ```roomAttributeUserOverride``` = {Attribute | Off}
-**Attribute** - Policy value allow **Voice identification** option on Microsoft Teams Room if transcription is started for the meeting.  
+**Attribute** - Policy value allow **Voice identification** option on Microsoft Teams Rooms if transcription is started for the meeting.  
 **Off** – No Voice identification option on Microsoft Teams Room. This is default.
```
Comment (JohanFreelancer9) [inline Teams/devices/smartvision-admin.md:177]:
```suggestion
**Off** – No Voice identification option on Microsoft Teams Rooms. This is the default.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the few additional suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tonysmit
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**